### PR TITLE
Update Accounts panel icons

### PR DESCRIPTION
**Pull request by GuillaumeRx**
[CH7360]

### CODE REVIEW

File: src/assets/icons/actions/edit.svg [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (FrederikBolding) [inline src/assets/icons/actions/edit.svg:1]: Remove height and width?

File: src/assets/icons/actions/add_bold.svg [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (FrederikBolding) [inline src/assets/icons/actions/add_bold.svg:1]: Remove width and height?

File: src/components/WalletUnlock/DeterministicAccountTable.tsx [hunk 3]
```diff
@@ -372,7 +372,7 @@ const DeterministicTable = ({
             onClick={generateFreshAddress}
             disabled={!isComplete || freshAddressIndex >= DEFAULT_GAP_TO_SCAN_FOR}
           >
-            <Icon type="add" color="none" />
+            <Icon type="add" color="none" width="32px" />
```
Comment (FrederikBolding) [inline src/components/WalletUnlock/DeterministicAccountTable.tsx:375]: This is not the same size as the blockies, is that intentional?

**Review by FrederikBolding — CHANGES_REQUESTED** [review]

File: src/components/WalletUnlock/DeterministicAccountTable.tsx (hunk 3 above)
Comment (GuillaumeRx) [reply src/components/WalletUnlock/DeterministicAccountTable.tsx:375]: I just removed the sizing on the svg, I put the value that was previously set in the svg's code. It was this size before :)

Comment (FrederikBolding) [reply src/components/WalletUnlock/DeterministicAccountTable.tsx:375]: I saw, just confirming that this is intentional.

Comment (GuillaumeRx) [reply src/components/WalletUnlock/DeterministicAccountTable.tsx:375]: Not really part of the work but I initially wanted to use the `add` icon - it was slightly different from the one I needed so I left the removal of the width/height but didn't use the icon :)

**Review by FrederikBolding — APPROVED** [review]

**Pull request merged**